updater.js

```diff
@@ -0,0 +1,225 @@
+var path = require('path')
+var fs = require('fs')
+var childProcess = require('child_process')
+var pkgUp = require('pkg-up')
+
+var BrowserslistError = require('./error')
+
+var PACKAGE_CANIUSE = 'caniuse-lite'
+
+function updateDB () {
+  var info = getMainInfo()
+  var lockfileRaw = fs.readFileSync(info.lockfile).toString()
+
+  var currentVersion = getCurrentVersion(lockfileRaw, info.packageManager)
+  var packageInfo = getLastVersionInfo()
+
+  console.log(
+    'Current version: ' + currentVersion + '\n' +
+    'New version: ' + packageInfo.version + '\n' +
+    'Updating ' + PACKAGE_CANIUSE + '…'
+  )
+
+  fs.writeFileSync(
+    info.lockfile,
+    updateLockfile(lockfileRaw, packageInfo, info.packageManager)
+  )
+  childProcess.execSync(info.packageManager + ' install')
+
+  console.log(PACKAGE_CANIUSE + ' has been successfully updated')
+}
+
+function getMainInfo () {
+  var packagePath = pkgUp.sync()
+  if (!packagePath) {
+    throw new BrowserslistError(
+      'Cannot find package.json. ' +
+      'Is it a right project to run npx browserslist --update-db?'
+    )
+  }
+
+  var rootDir = path.dirname(packagePath)
+
+  var packageManager = ''
+  var lockfile = ''
+
+  var lockfileNpm = path.join(rootDir, 'package-lock.json')
+  var lockfileYarn = path.join(rootDir, 'yarn.lock')
+  var lockfilePnpm = path.join(rootDir, 'pnpm-lock.yaml')
+
+  if (fs.existsSync(lockfileNpm)) {
+    packageManager = 'npm'
+    lockfile = lockfileNpm
+  } else if (fs.existsSync(lockfileYarn)) {
+    packageManager = 'yarn'
+    lockfile = lockfileYarn
+  } else if (fs.existsSync(lockfilePnpm)) {
+    packageManager = 'pnpm'
+    lockfile = lockfilePnpm
+  } else {
+    throw new BrowserslistError('Lockfile search error')
+  }
+
+  return {
+    packageManager: packageManager,
+    lockfile: lockfile
+  }
+}
+
+function getCurrentVersion (lockfileRaw, packageManager) {
+  if (packageManager === 'npm') {
+    var parsedFile = JSON.parse(lockfileRaw)
+    if (
+      parsedFile.dependencies !== undefined &&
+      parsedFile.dependencies[PACKAGE_CANIUSE] !== undefined &&
+      parsedFile.dependencies[PACKAGE_CANIUSE].version !== undefined
+    ) {
+      return parsedFile.dependencies[PACKAGE_CANIUSE].version
+    }
+    return null
```

Comment:
`Current version is null` will be bad output.

I think you need better message for this case.